internal/c99/headers/windows_amd64/builtin.h

```diff
@@ -0,0 +1,137 @@
+//  Copyright 2017 The CCIR Authors. All rights reserved.
+//  Use of this source code is governed by a BSD-style
+//  license that can be found in the LICENSE file.
+
+// +build ignore
+
+#ifndef _BUILTIN_H_
+#define _BUILTIN_H_
+
+#include "predefined.h"
+
+typedef struct {
+	void *_[2];		// Go *[]interface{}
+} *__builtin_va_list;
+
+#define _MSC_VER 1900
+#define _MSVCRT_
+#define _MSC_EXTENSIONS
+#define __int8 char
+#define BYTE unsigned char
+#define __int16 short
+#define __int32 int
+#define __int64 long long
+// disable SSE and other instrisics we dont support
+#define _X86INTRIN_H_INCLUDED 1
+#undef __SSE__
+#undef __SSE2__
+#undef __MMX__
+// bypass bug that fails for: #define MICROSOFT_WINDOWS_WINBASE_H_DEFINE_INTERLOCKED_CPLUSPLUS_OVERLOADS (_WIN32_WINNT >= 0x0502 || !defined (_WINBASE_))
```

Comment:
Assuming the bug is probably in c99/cpp.go: Can you please add a small repro case in either c99/testdata/cpp-expand or c99/testdata/bug/nn.c if the bug is elsewhere?

Reply:
reproduction in https://github.com/cznic/sqlite2go/pull/7/commits/009e2de130daf943a72d3af25a131bf0530731f6